.github/workflows/build-and-run-node-test.yaml

```
@@ -0,0 +1,39 @@
+---
+name: "Build and test"
+on:
+  push:
+    paths:
+      - "node/**"
+      - "crates/**"
+      - "pallets/*"
+      - "runtime/*"
```

Ah, okay I didn't read this carefully enough then. Nice that it doesn't do the full build for things like the `README`, that's something that annoyed me about the Circle CI config.

Let's change this to the following, since we have some nested folders in `pallets` and `runtime` where we'd want to run the CI if they changed (e.g `runtime/src/weights`):

```suggestion
name: "Build and test"
on:
  push:
    paths:
      - "node/**"
      - "crates/**"
      - "pallets/**"
      - "runtime/**"
```